src/cmd/uds.go

```diff
@@ -56,6 +56,16 @@ var deployCmd = &cobra.Command{
 		bundleCfg.DeployOpts.Source = chooseBundle(args)
 		configureZarf()
 
+		// load uds-config if it exists
+		config := v.ConfigFileUsed()
+		if config != "" {
+			bundleCfg.DeployOpts.Config = config
+			if err := loadViperConfig(); err != nil {
```

Comment:
did we remove the `loadViperConfig()` call from here in a different PR?

Reply:
Yes, it was refactored in https://github.com/defenseunicorns/uds-cli/pull/618 and we'll follow up in https://github.com/defenseunicorns/uds-cli/issues/641